tools/sctrace/README.md

```diff
@@ -0,0 +1,357 @@
+# Syscall-Compliance-Trace
+
+Syscall-Compliance-Trace (sctrace) is a powerful system call compliance
+verification tool that analyzes and validates system call against user-defined
+compliance patterns. Written in SCML (System Call Matching Language), these
+patterns describe supported functionality of system calls.
+sctrace supports both real-time monitoring of running programs and post-analysis of
+existing trace logs, providing comprehensive insights into system call compliance
+with intuitive pattern matching and visual feedback.
+
+## Features
+
+- **Pattern-based filtering**: Define system call patterns using SCML syntax
+- **Dual mode operation**:
+  - Online mode: Real-time tracing of running programs
+  - Offline mode: Analysis of existing strace log files
+- **Quiet mode**: Minimal output showing only unsupported calls
+- **Multi-threaded support**: Automatic handling of multi-threaded program traces with syscall reconstruction
+
+## Installation
+
+Make sure you have Rust installed, then build the project:
+
+```bash
+cargo build --release
+```
+
+The binary will be available at `target/release/sctrace`.
+
+## Prerequisites
+
+- **strace** version 5.15 or higher (for online mode)
```

Comment:
Give a link to `strace` and the command to install strace. In addition, we should preinstall strace in Asterinas Docker image.